spl/token/instructions.py

```diff
@@ -918,3 +939,41 @@ def burn2(params: Burn2Params) -> TransactionInstruction:
         dict(instruction_type=InstructionType.BURN2, args=dict(amount=params.amount, decimals=params.decimals))
     )
     return __burn_instruction(params, data)
+
+
+def get_associated_token_address(params: AssociatedTokenAccountParams) -> PublicKey:
```

Comment:
Can we re-name this to `find_associated_token_address` so it matches the Solana docs?

Reply:
@michaelhly  in the source is where I got that function name from: https://github.com/project-serum/serum-ts/blob/master/packages/associated-token/src/index.ts#L22

Reply:
Ah, you're right. It's also get [here](https://github.com/solana-labs/solana-program-library/blob/3dd67672974f92d3b648bb50ee74f4747a5f8973/token/js/client/token.js#L2236).